src/components/Utils/orderResource.js

```diff
@@ -17,26 +15,33 @@ const saveOrder = (order, mutator) => {
   return method(order);
 };
 
+export const getFullOrderNumber = ({ poNumberPrefix = '', poNumberSuffix = '', poNumber = '' }) => (
+  `${poNumberPrefix}${poNumber}${poNumberSuffix}`.trim()
+);
+
 export const updateOrderResource = (order, mutator, changedProps) => {
   const clonedOrder = cloneDeep(order);
 
+  clonedOrder.poNumber = getFullOrderNumber(clonedOrder);
+
   Object.assign(clonedOrder, changedProps);
 
   return saveOrder(clonedOrder, mutator);
 };
 
 export const createOrderResource = (order, mutator) => {
   const clonedOrder = cloneDeep(order);
-  const { numberPrefix = '', numberSuffix = '', poNumber = '' } = clonedOrder;
-  const fullOrderNumber = `${numberPrefix}${poNumber}${numberSuffix}`.trim();
 
-  clonedOrder.poNumber = fullOrderNumber || undefined;
+  clonedOrder.poNumber = getFullOrderNumber(clonedOrder) || undefined;
```

Comment:
have you checked with the back-end if we still need this?

Reply:
The story description provides it: 
![image](https://user-images.githubusercontent.com/43407139/80001701-59501400-84c7-11ea-8aab-d3e370f04f7b.png)